beacon-chain/operations/voluntaryexits/service.go

```diff
@@ -34,7 +36,10 @@ func NewPool() *Pool {
 func (p *Pool) PendingExits(state *beaconstate.BeaconState, slot uint64) []*ethpb.SignedVoluntaryExit {
 	p.lock.RLock()
 	defer p.lock.RUnlock()
-	pending := make([]*ethpb.SignedVoluntaryExit, 0)
+
+	// Allocate pending slice with a capacity of min(len(p.pending), maxVoluntaryExits) since the
+	// array cannot exceed the max and is typically less than the max value.
+	pending := make([]*ethpb.SignedVoluntaryExit, 0, mathutil.Max(uint64(len(p.pending)), params.BeaconConfig().MaxVoluntaryExits))
```

Comment:
This is right. It uses max

Reply:
This is wrong, it should be min. I’ll fix

Reply:
Done!